### DISCUSSION TITLE
primary requiv not changing

### DISCUSSION THREAD

**Discussion by kathmcp** [phoebe-project/phoebe2#494] [General]
I am using a python interface, via jupyter notebook, and in trying to set up my contact binary, i keep getting numerous warnings. I assume my starting parameters are not close enough to a solution, which is causing the problem. But when I update requiv for primary and rerun the code, the warning doesn't seem to know I have updated requiv.
![ErrorRequiv](https://user-images.githubusercontent.com/81473857/125768165-524dd631-f5f4-4636-8866-5df236123536.PNG)

**Comment by kecnry** [comment]
Those are most likely intermediate warnings.  The warnings are checked after _each value_ is changed by default, and so likely are being raised when you change `sma` but before you change `requiv`.  These automated warnings can be disabled by calling [phoebe.interactive_checks_off()](http://phoebe-project.org/docs/latest/api/phoebe.interactive_checks_off) (which is slated to become the new default in the 2.4 release as more people are using jupyter and this is a common point of confusion).

You can manually run any checks with [print(b.run_checks())](http://phoebe-project.org/docs/latest/api/phoebe.frontend.bundle.Bundle.run_checks).  If any checks are not passing, then they will be raised as errors once you call `run_compute`.  You can also always print the value of the primary radius to confirm it has been set properly: `print(b_cb.get_value('requiv', component='primary'))`.
